plan/plan.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/pingcap/tidb/expression"
 	"github.com/pingcap/tidb/sessionctx"
 	"github.com/pingcap/tidb/util/codec"
-	tipb "github.com/pingcap/tipb/go-tipb"
+	"github.com/pingcap/tipb/go-tipb"
```

Comment:
why make this change ?

Reply:
It seems that my local `goimports` does this change.

Reply:
maybe we should revert this change, to be consistent with other import codes.